android/src/main/java/com/pylon/RNSpokestack/RNSpokestackModule.java

```diff
@@ -101,6 +77,16 @@ public void stop () {
     pipeline.stop();
   }
 
+  @ReactMethod
+  public void activate () {
```

Comment:
smart to make these methods, because you may want to trigger the activate event someday based on a manual activation

Reply:
neato

Reply:
After thinking about this again, I think it would be good to raise the activate/deactivate events on the context in these methods. The app may want to trigger visuals based on the activation state, and this triggering would be the same for automatic and manual activations. This can be done with the following:

```java
pipeline.getContext().dispatch(SpeechContext.Event.ACTIVATE);
```